search.py

```diff
@@ -474,14 +453,13 @@ def handle_search_task(task: tasks.Task) -> bool:
 
 
 # @bpy.app.handlers.persistent
-def search_timer():
+def startup_search_timer():
   # this makes a first search after opening blender. showing latest assets.
   # utils.p('timer search')
   # utils.p('start search timer')
   global first_time
   preferences = bpy.context.preferences.addons['blenderkit'].preferences
-  if first_time and not bpy.app.background:  # first time
-
+  if first_time and not bpy.app.background and global_vars.DAEMON_ONLINE:  # first time
```

Comment:
I am wondering here, if tooltip or even updater needs to wait for DAEMON_ONLINE?